src/pages/watch-later/watch-later.js

```diff
@@ -1,8 +1,18 @@
 import styles from "./watch-later.module.scss"
+import { VideoCard, SidePane } from "@components"
 
 const WatchLater = () => {
     return (
-        <div>WatchLater</div>
+        <div className={styles.container}>
+            <SidePane title="Watch Later..." time="30 Minutes" count={30} />
+            <div className={styles.main}>
+                <VideoCard orientation="horizontal" />
+                <VideoCard orientation="horizontal" />
+                <VideoCard orientation="horizontal" />
+                <VideoCard orientation="horizontal" />
+                <VideoCard orientation="horizontal" />
```

Comment:
Since this is a basic layout-it's fine, later you can map through the repeating component.